plugin/manifest.yaml

```diff
@@ -104,23 +104,6 @@ data:
     }
 ---
 apiVersion: v1
-kind: ConfigMap
```

Comment:
since this isn't needed today, it doesn't have to be created in the dev environment. But it doesn't hurt to create one that has empty data, just as a placeholder for future enhancement. Or comment this out. But I would leave it here in some state (commented out or not) just so in the future when we need to add configuration, we can just resurrect this without worrying about how it needs to be done. The work is already done here.